### ISSUE TITLE
operator_usage_whitespace doesn't trigger if operand has parentheses

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#989]
Examples:

```swift
let v8 = Int8(1)  << 6
let v8 = 1 <<  (6)
```
